deblur/test/test_workflow.py

```diff
@@ -196,7 +188,7 @@ def test_remove_artifacts_seqs(self):
                         "TCGGCTTTGTAAATCCCTGGGTAAATAGGGT"),
                ("ref6", "TACCCGCAGCTCAAGTGGTGGTCGCTATTATTGAGCCTAAAACGTCCGTAG"
                         "TCGGCTTTGTAAATCCCTGGGTAAATCGGGT")]
-        ref_fp = join(self.working_dir, "ref.fasta")
+        ref_fp = join(self.working_dir, "ref2.fasta")
```

Comment:
Does this mean that those files where never removed? Note that the tests should clean up the environment after running the tests - so ref_fp should be added to the `files_to_remove` list.